python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -101,13 +99,19 @@ def __init__(
             # If only a mode was passed, use anonymous metric
             self.default_metric = DEFAULT_METRIC
 
-        (
-            self._fs,
-            self._experiment_fs_path,
-            self._experiment_json_fs_path,
-        ) = self._get_experiment_fs_and_path(
-            experiment_checkpoint_path, storage_filesystem
-        )
+        self._fs = storage_filesystem
```

Comment:
This actually does need to handle `storage_filesystem=None` for usage outside of `tune.run`: `ExperimentAnalysis("s3://...")`

So, we can just use `ray.train._internal.storage.get_fs_and_path` if no storage filesystem is provided.

Reply:
Hm, I don't think we should support this API. ExperimentAnalysis is deprecated right?

Reply:
I see this is marked as PublicAPI still, so reverted the change.